src/flex/agentApp/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the MIT License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+module.exports = (config, { isProd, isDev, isTest }) => {
+  /**
```

Comment:
Use `//` line comments when just commenting on executable code and remove the extra blank line below.